_posts/2022-03-28-embed-google-webfonts.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Embedding Google Fonts for better Privacy and GDPR conformance
+author: esko.suomi
+excerpt: >
+  Regional court in Munich, Germany recently rules Google Fonts as non-GDPR compliant due to the serving mechanism. This does pose a minor challenge, how does one embed webfonts to stay GDPR compliant?
+
+tags:
+ - Google Fonts
+ - GDPR
+ - Privacy
+---
+
+Recently the regional court in Germany in Munich declared that embedding Google Fonts on a website violates GDPR ([news article](https://rewis.io/urteile/urteil/lhm-20-01-2022-3-o-1749320/), [judgement](https://rewis.io/s/u/zH2/)).
```

Comment:
judgement (U.K.) vs judgment - can use either, but must be consistent. U.S. english is kinda the default for blogs, in that case judgment would be right. If intention is to use U.K. english then it's good as it is, just need to make a consistent choice.